frontend/src/index.js

```diff
@@ -17,17 +17,15 @@ require('./css/index.styl')
 require('./util/appInterface.js')
 require('./util/tinymceInit.js')
 
-// export const history = require('history').createBrowserHistory()
-
 ReactDOM.render(
   <Provider store={store}>
-    <Router history={history}>
+    <BrowserRouter history={history}>
```

Comment:
I'm not sure that I understood, at `frontend_app_admin_workspace_user/src/index.js` you add the comment `// INFO - CH - 2020-01-11 - User Router instead of BrowserRouter because BrowserRouter doesn't have a history props` but here you use BrowserRouter and history.

Reply:
it's a mistake, I'm fixing it.
Only frontend must use BrowserRouter actually. Because apps needs to get their history from the frontend, so they must use Router

Reply:
actually, there might be a small misconception here

Reply:
ok the issue was in frontend, it was using a BrowserRouter with a history props, which makes no sense.

Reply:
fixed